logstash-core/lib/logstash/java_pipeline.rb

```diff
@@ -92,71 +88,8 @@ def compile_lir
     LogStash::Compiler.compile_sources(sources_with_metadata, @settings)
   end
 
-  def buildOutput(name, line, column, *args)
-    plugin("output", name, line, column, *args)
-  end
-
-  def buildFilter(name, line, column, *args)
-    plugin("filter", name, line, column, *args)
-  end
-
-  def buildInput(name, line, column, *args)
-    plugin("input", name, line, column, *args)
-  end
-
-  def buildCodec(name, *args)
-   plugin("codec", name, 0, 0, *args)
-  end
-
   def plugin(plugin_type, name, line, column, *args)
-    @plugin_counter += 1
```

Comment:
was this a dead variable ?

Reply:
@jakelandis because there's no one reading this variable (as seen by the fact that removing it doesn't lead to issues).